index.html

```diff
@@ -18,55 +18,78 @@
 
         <h3 class="logo"><a href="#">My Logo</a></h3>
 
-        <nav>
+        <nav class="nav1">
 
             <img src="./img/hamburger.png" alt="hamburger_image">
 
         </nav>
 
+        <nav class="nav2">
+
+            <ul class="nav2 liststyle">
+                <li class="list_style"> <a class="textdecoration" href="#">Portfolio</a></li>
+                <li class="list_style"> <a class="textdecoration" href="#">About</a></li>
+                <li class="list_style"><a class="textdecoration" href="#">Contact</a></li>
+            </ul>
+
+
+        </nav>
+
     </header>
 
     <section class="section1">
```

Comment:
https://github.com/EmekaEmmanuel/portfolio_mobile_day3/blob/d910d203c2003cf8d6a80ff48513ee9ef14b030e/index.css#L32-L39

Please update your `.section1` to have the corresponding background-image in the desktop version 😉:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/199864349-260d7c24-5802-4064-ae76-bf797e79103e.png) | ![image](https://user-images.githubusercontent.com/42799579/199864394-a7598495-d4c8-475a-8c82-7d82e65e9820.png)  |

> Notice how your background-image is larger, looks like it is zoomed in.